packages/inter-protocol/src/proposals/utils.js

```diff
@@ -8,13 +8,13 @@ export const DEPOSIT_FACET = 'depositFacet';
 const { details: X } = assert;
 
 /**
- * @param {ERef<NameAdmin>} nameAdmin
+ * @param {ERef<import('@agoric/vats/src/nameHub.js').NameAdmin>} nameAdmin
```

Comment:
How hard would it be to export types at the root of the `vats` package?

Reply:
good call. in the course of the PR they were exported but these imports needed updating.